worlds/sc2/docs/en_Starcraft 2.md

```diff
@@ -18,50 +20,74 @@ You find items by making progress in these categories:
 * Reaching milestones in the mission, such as completing part of a main objective
 * Completing challenges based on achievements in the base game, such as clearing all Zerg on Devil's Playground
 
-Except for mission completion, these categories can be disabled in the game's settings. For instance, you can disable getting items for reaching required milestones.
+Except for mission completion, these categories can be disabled in the game's settings. 
```

Comment:
> Except for mission completion, these categories can be disabled in the game's settings.

This is a little difficult to understand, as "these" is not really defined to a new player.  Instead, I might recommend:

> Other than mission completion locations, various location categories can be disabled in the player options.

I recommend using Options over Settings as the Archipelago terminology changed around this in the last year to avoid confusion with host settings vs player options.

Reply:
Setting was changed to option where applicable.
I modified a little your proposal. You are welcome to make sure that it is good!

> Aside from mission completion, the other location categories can be disabled in the player options.